cpp/src/strings/regex/regcomp.cpp

```diff
@@ -1100,15 +1086,16 @@ void reprog::print(regex_flags const flags)
     const reclass& cls = _classes[i];
     auto const size    = static_cast<int>(cls.literals.size());
     printf("%2d: ", i);
-    for (int j = 0; j < size; j += 2) {
-      char32_t c1 = cls.literals[j];
-      char32_t c2 = cls.literals[j + 1];
+    for (int j = 0; j < size; ++j) {
```

Comment:
The change to use `reclass_range` makes this more obvious than before. I like this change.